streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -1844,19 +1844,17 @@ public void process(final Object key, final Object value) {}
         assertEquals(0, punctuatedWallClockTime.size());
 
         mockTime.sleep(100L);
-        for (long i = 0L; i < 10L; i++) {
```

Comment:
Why do we remove this loop?

Reply:
Since we do not need it really: a single record is sufficient to trigger the punctuation.